labeled_data/technology/ai/generative_ai/application/code.yml

```diff
@@ -1,5 +1,5 @@
 name: Artificial Intelligence - application - code
```

Comment:
The name of this label should be `Artificial Intelligence - Application - Code`

Reply:
Good. About to use `Artificial Intelligence - Generative Artificial Intelligence - Application - Code`. Reference discussion 
 https://github.com/X-lab2017/open-digger/pull/1487#discussion_r1448367064.